Client/qtTeamTalk/preferences.ui

```diff
@@ -1576,7 +1576,7 @@
      </widget>
      <widget class="QWidget" name="ttsTab">
       <attribute name="title">
-       <string>Text To Speech</string>
+       <string>Notification</string>
```

Comment:
How come you want to call it "Notification" instead of "Text To Speech". "Sound Events" are also sort of "Notification".